extensions/adobe/experience/primetime-authentication/authorization.schema.json

```diff
@@ -0,0 +1,113 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":   "https://ns.adobe.com/experience/primetime-authentication/authorization",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Authorization details",
+  "type": "object",
+  "description":
+    "Authorization details properties specific to Adobe Primetime Authentication flows.",
+  "definitions": {
+    "rational": {
+      "$id": "https://ns.adobe.com/experience/primetime-authentication/authorization/resource",
+      "title": "Resource",
+      "description":
+        "A representation of the media resource the end user wants to access.",
+      "properties": {
+        "xdm:channel": {
+          "title": "Resource channel",
+          "description": "The resource channel",
+          "type": "string"
+        },
+        "xdm:raw": {
+          "title": "Raw resource",
+          "description": "The raw representation of the resource",
+          "type": "string"
+        },
+        "xdm:authorized" : {
+          "title" : "Authorization status",
+          "description": "Authorization decision for each resource",
+          "type": "boolean"
+        }
+      },
+      "required": ["xdm:channel", "xdm:authorized"]
+    },
+    "authorization": {
+      "properties": {
+        "xdm:idp": {
+          "title": "MVPD Identifier",
+          "type": "string",
+          "description":
+            "The MVPD with whom the end user is authorized or tries to authorize"
+        },
+        "xdm:technologyProvider": {
+          "title": "Technology Provider",
+          "type": "string",
+          "description":
+            "The partner which provides the infrastructure for IdP authorization endpoints"
+        },
+        "xdm:broker": {
+          "title": "Authorization Broker",
+          "type": "string",
+          "description":
+            "External authorization provider which might intermediate the authorization flow"
+        },
+        "xdm:authorizationType": {
+          "title" : "Authorization Type",
+          "type": "string",
+          "description": "The authorization type",
+          "meta:enum": {
+            "authoritative" : "The flow through which the end user obtains an authoritative authorization decision.",
+            "non-authoritative" : "The flow through which the end user obtains a non-authoritative authorization decision useful initial content filtering"
```

Comment:
Please add **for** between `authorization decision useful ` and ` initial content filtering`.

Reply:
Done.